testsuite/interrupt_windows.go

```diff
@@ -61,5 +61,17 @@ func sendInterrupt(process *os.Process) error {
 	if r1 == 0 {
 		return err
 	}
+
+	// Free the console after sending the interrupt
```

Comment:
Interesting. I did some reading. If easy enough, can you try to remove this new code and the code that does the `AttachConsole` and `SetConsoleCtrlHandler`? Basically, try to make this entire function be:

```go
func sendInterrupt(process *os.Process) error {
	dll, err := syscall.LoadDLL("kernel32.dll")
	if err != nil {
		return err
	}
	f, err := d.FindProc("GenerateConsoleCtrlEvent")
	if err != nil {
		return err
	}
	r, _, e := p.Call(uintptr(syscall.CTRL_BREAK_EVENT), uintptr(process.Pid))
	if r == 0 {
		return err
	}
	return nil
}
```

And see if that's good enough. Also note I moved to [`syscall`](https://pkg.go.dev/syscall?GOOS=windows) instead of `golang.org/x/sys/windows` dependency.

Reply:
this doesn't seem to work yet, debugging. syscall.LoadDLL is deprecated since go 1.8, debugging with windows.LoadDLL atm

Reply:
this still sends CTRL_BREAK_EVENT to all processes that share the console

Reply:
one alternative option would be start the dev server in a new console with on Windows with
```
cmd.SysProcAttr = &syscall.SysProcAttr{
    CreationFlags: syscall.CREATE_NEW_CONSOLE,
}
```

here https://github.com/temporalio/sdk-go/blob/1a0d5f99f30134236c60c45d89fa909600ff451a/testsuite/devserver.go#L112

Reply:
or both on Windows and unix